docs/content/examples/tutorials/docker-build.md

```diff
@@ -12,11 +14,33 @@ docker build -t mailserver/docker-mailserver .
 ```
 
 Or, you can clone and retrieve the submodules in one command:
+
 ```sh
 git clone --recurse-submodules https://github.com/docker-mailserver/docker-mailserver
 ```
 
 Retrieving the git submodules will fix the error:
-```
+
+```txt
 COPY failed: file not found in build context or excluded by .dockerignore: stat target/docker-configomat/configomat.sh: file does not exist
 ```
+
+### About Docker
+
+#### Version
+
+We make use of build-features that require a recent version of Docker. Depending on your distribution, please have a look at [the official installation documentation for Docker](https://docs.docker.com/engine/install/) to get the latest version. Otherwise, you may encounter issues, for example with the `--link` flag for a `#!dockerfile COPY` command.
```

Comment:
`#!dockerfile COPY`

Is `#!dockerfile` a typo?

```suggestion
We make use of build-features that require a recent version of Docker. Depending on your distribution, please have a look at [the official installation documentation for Docker](https://docs.docker.com/engine/install/) to get the latest version. Otherwise, you may encounter issues, for example with the `--link` flag for a Dockerfile [COPY](https://docs.docker.com/engine/reference/builder/#copy) command.
```

Reply:
No, not a type: <https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlighting-inline-code-blocks>.

But we can also provide the link you mentioned. I think this is a good all-in-one solution.

Reply:
resolved in [b78deb8](https://github.com/docker-mailserver/docker-mailserver/pull/2799/commits/b78deb8ccd0b3cbe5e9ee283c3475ee2642112a2)